### PR TITLE
Era changing

### PR DESCRIPTION
**Pull request by tylermcdonald**

### CODE REVIEW

File: app/environments/brassbirmingham/brassbirmingham/envs/classes/board.py [hunk 1]
```diff
@@ -500,21 +507,65 @@ def sellBuilding(self, building: MarketBuilding, player: Player):
         self.removeXBeer(building.beerCost, [building.town], player)
         building.sell()
 
-    def getVictoryPoints(self) -> Dict[str, int]:
-        points = {player.id: player.victoryPoints for player in self.players}
+    def getVictoryPoints(self) -> Dict[Player, int]:
+        points = {player: player.victoryPoints for player in self.players}
```
Comment (noahpodgurski) [inline app/environments/brassbirmingham/brassbirmingham/envs/classes/board.py:511]: I still feel like just adding a `victoryPoint` attr to each `Player` object is a better solution here

Comment (tylermcdonald) [reply app/environments/brassbirmingham/brassbirmingham/envs/classes/board.py:511]: How would that work? VPs are only incremented at end of eras

**Review by noahpodgurski — APPROVED** [review]
looks good

File: app/environments/brassbirmingham/brassbirmingham/envs/classes/board.py (hunk 1 above)
Comment (noahpodgurski) [reply app/environments/brassbirmingham/brassbirmingham/envs/classes/board.py:511]: I guess that's true but I feel like the data structure makes more sense that way idk not a big deal

**Pull request merged**